docs/ensnode.io/src/components/organisms/ExamplesPage.tsx

```diff
@@ -0,0 +1,309 @@
+                  className="w-full mt-4 px-3 py-2 text-sm text-gray-600 hover:text-gray-800 border border-gray-300 rounded-md hover:bg-gray-50 transition-colors"
+                >
+                  Clear Filter
+                </button>
+              )}
+            </div>
+          </div>
+
+          <div className="flex-1">
+            <div className="mb-6">
+              <p className="text-sm text-gray-600">
+                {filteredExamples.length === examples.length ? (
+                  <>Showing all {filteredExamples.length} queries</>
+                ) : (
+                  <>
+                    Showing {filteredExamples.length} of {examples.length} queries
+                    {selectedCategory && (
+                      <span className="ml-2 inline-flex items-center px-2.5 py-0.5 rounded-full text-xs font-medium bg-blue-100 text-[var(--color-primary)]">
+                        {selectedCategory}
+                      </span>
+                    )}
+                  </>
+                )}
+              </p>
+            </div>
+
+            {filteredExamples.length > 0 ? (
+              <div className="grid gap-6 md:grid-cols-2 xl:grid-cols-3">
+                {filteredExamples.map((example) => (
+                  <QueryCard key={example.id} example={example} />
+                ))}
+              </div>
+            ) : (
+              <div className="text-center py-12">
+                <svg
+                  className="mx-auto h-12 w-12 text-gray-400"
+                  fill="none"
+                  viewBox="0 0 24 24"
+                  stroke="currentColor"
+                >
+                  <path
+                    strokeLinecap="round"
+                    strokeLinejoin="round"
+                    strokeWidth="2"
+                    d="M9.172 16.172a4 4 0 015.656 0M9 12h6m-6-4h6m2 5.291A7.962 7.962 0 0112 15c-2.34 0-4.5-.9-6.172-2.172M15 21H3a2 2 0 01-2-2V5a2 2 0 012-2h12a2 2 0 012 2v16a2 2 0 01-2 2z"
+                  />
+                </svg>
+                <h3 className="mt-2 text-sm font-medium text-gray-900">No queries found</h3>
+                <p className="mt-1 text-sm text-gray-500">
+                  Try adjusting your search or filter criteria.
+                </p>
+              </div>
+            )}
+          </div>
+        </div>
+      </div>
+    </div>
+  );
+};
+
+interface QueryCardProps {
+  example: SavedQuery;
+}
+
+const QueryCard: React.FC<QueryCardProps> = ({ example }) => {
+  const slugify = (text: string): string => {
+    return text
+      .toLowerCase()
+      .replace(/[^a-z0-9]+/g, "-")
+      .replace(/(^-|-$)/g, "");
+  };
+
+  const handleInspect = () => {
+    const slug = slugify(example.name);
+    window.location.href = `/examples/${slug}`;
+  };
+
+  const createGraphiQLUrl = (query: string, variables: string): string => {
+    try {
+      const baseUrl = "https://admin.ensnode.io/gql/subgraph-compat";
```

Comment:
I haven't really given this function much thought, yes there's some duplication, and yes the URL is hard coded, but it would be great to avoid adding environment variables for this since the URL is unlikely to ever change.